experimental/sev_guest/src/io.rs

```diff
@@ -165,43 +166,118 @@ where
 /// Factory for creating port reader and writers that perform direct raw IO operations.
```

Comment:
Do we mean the same by "direct" and "raw"? If so, it we might stick to one of them. (also below)

Reply:
Direct and raw imply roughly the same thing. I updated the comment to be consistent with the usage throughout the rest of the file: "raw" refers to the port implementation that does "direct" IO.